test/unit/submission/validation.spec.ts

```diff
@@ -1444,6 +1445,98 @@ describe('data-validator', () => {
 
       chai.expect(result.specimen.dataErrors).to.deep.include(specimenMissingDonorAB2FieldErr);
     });
+    it('should detect forbidden provided fields', async () => {
```

Comment:
Haha, this sounds like your testing the value type of the record fields. Maybe something more like `should detect forbidden provided fields in submission record for registered donor normal specimen`